consensus/polybft/validator_set.go

```diff
@@ -97,6 +91,10 @@ func (vs validatorSet) Len() int {
 	return vs.validators.Len()
 }
 
-func (vs validatorSet) getQuorumSize() int64 {
-	return int64(math.Ceil((2 * float64(vs.totalVotingPower)) / 3))
+// getQuorumSize calculates quorum size as 2/3 super-majority of provided total voting power
+func getQuorumSize(totalVotingPower *big.Int) *big.Int {
+	quorum := new(big.Int)
+	quorum.Mul(totalVotingPower, big.NewInt(2))
+	// divide and round up result to next bigger integer value
+	return quorum.Add(quorum, big.NewInt(3)).Sub(quorum, big.NewInt(1)).Div(quorum, big.NewInt(3))
```

Comment:
Isn't this the same?: 
```suggestion
	return quorum.Add(quorum, big.NewInt(2)).Div(quorum, big.NewInt(3))
```